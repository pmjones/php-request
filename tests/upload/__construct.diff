002+ fail reconstruct
002- ok reconstruct