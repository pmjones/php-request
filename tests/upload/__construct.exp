string(10) "SapiUpload"
ok reconstruct