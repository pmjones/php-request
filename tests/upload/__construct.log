
---- EXPECTED OUTPUT
string(10) "SapiUpload"
ok reconstruct
---- ACTUAL OUTPUT
string(10) "SapiUpload"
fail reconstruct
---- FAILED
