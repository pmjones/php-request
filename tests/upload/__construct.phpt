--TEST--
SapiUpload::__construct
--FILE--
<?php

// Basic construct
$upload = new SapiUpload('foo', 'text/plain', 123, '/tmp/foo.txt', 0);
var_dump(get_class($upload));

// Check __construct can't be called twice
try {
    $upload->__construct(null, null, null, null, null);
    echo 'fail reconstruct' . PHP_EOL;
} catch( RuntimeException $e ) {
    echo 'ok reconstruct' . PHP_EOL;
}

--EXPECT--
string(10) "SapiUpload"
ok reconstruct
