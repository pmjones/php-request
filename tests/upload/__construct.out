string(10) "SapiUpload"
fail reconstruct